adapters/Mintegral/MintegralAdapter/Bidding/GADMAdapterMintegralBannerRenderer.m

```diff
@@ -61,25 +60,18 @@ - (void)renderBannerForAdConfiguration:(nonnull GADMediationBannerAdConfiguratio
         return delegate;
     };
         
-    NSString *unitId = adConfiguration.credentials.settings[GADMAdapterMintegralAdUnitID];
+    NSString *adUnitId = adConfiguration.credentials.settings[GADMAdapterMintegralAdUnitID];
     NSString *placementId = adConfiguration.credentials.settings[GADMAdapterMintegralPlacementID];
     UIViewController *rootViewController = adConfiguration.topViewController;
 
-    if ([GADMAdapterMintegralUtils isEmpty:unitId]) {
-        NSError *error =
-        GADMAdapterMintegralErrorWithCodeAndDescription(GADMintegralErrorInvalidServerParameters, @"Unit ID cannot be nil.");
-        _adLoadCompletionHandler(nil, error);
-        return;
-    }
-    
-    if ([GADMAdapterMintegralUtils isEmpty:adConfiguration.bidResponse]) {
+    if ([GADMAdapterMintegralUtils isStringEmpty:adUnitId]) {
         NSError *error =
-        GADMAdapterMintegralErrorWithCodeAndDescription(GADMintegralErrorInvalidServerParameters, @"bid token cannot be nil.");
+        GADMAdapterMintegralErrorWithCodeAndDescription(GADMintegralErrorInvalidServerParameters, @"Ad Unit ID connot be nil.");
         _adLoadCompletionHandler(nil, error);
         return;
     }
     
-    _bannerAdView = [[MTGBannerAdView alloc]initBannerAdViewWithAdSize:adConfiguration.adSize.size placementId:placementId unitId:unitId rootViewController:rootViewController];
+    _bannerAdView = [[MTGBannerAdView alloc]initBannerAdViewWithAdSize:adConfiguration.adSize.size placementId:placementId unitId:adUnitId rootViewController:rootViewController];
```

Comment:
Does Mintegral SDK handle invalid banner size? 

We should map the AdMob banner size to Mintegral banner size, and check if the banner size is valid before trying to load a banner ad, see [this example](https://github.com/googleads/googleads-mobile-ios-mediation/blob/main/adapters/Pangle/PangleAdapter/Bidding/GADPangleRTBBannerRenderer.m#L69).

Reply:
Gentle bump on this command.

Reply:
@Mintegral-Lee Is there an update on this?